projects.json

```diff
@@ -3724,5 +3724,84 @@
         "action": "TestSwiftPackage"
       }
     ]
+  },
+  {
+    "repository": "Git",
+    "url": "https://github.com/raywenderlich/vapor-til.git",
```

Comment:
At this point we can only accept projects with these specific licenses:
BSD
MIT
Apache License, version 2.0
Eclipse Public License
Mozilla Public License (MPL) 1.1
MPL 2.0
CDDL

Vapor TIL doesn't appear to have one of these licenses from what I can tell. SteamPress and SteamPressExample look fine, but this will need to be removed from the pull request for now.

Reply:
Ah yeah apologies, forgot it's not permissively licenced. Removed